src/types/FireOrBufferWithNames.ts

```diff
@@ -0,0 +1,6 @@
+type FileOrBufferWithNames = {
+  file: Buffer | File;
+  name: string;
+};
+
+export default FileOrBufferWithNames;
```

Comment:
Don't do any `default` exports, it causes issues when exporting the type at the module level. Instead just an `export` is fine.